operating/use-admission-controller-enforcement.adoc

```diff
@@ -0,0 +1,65 @@
+[id="use-admission-controller-enforcement"]
+= Using admission controller enforcement
+include::modules/common-attributes.adoc[]
+:context: use-admission-controller-enforcement
+
+toc::[]
+
+[role="_abstract"]
+{product-title} works with https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/[Kubernetes Admission Controllers] and https://docs.openshift.com/container-platform/4.5/architecture/admission-plug-ins.html[{ocp} Admission plug-ins] to allow you to enforce security policies before Kubernetes or {ocp} creates workloads, for example, deployments, daemon sets or jobs.
+{product-title} admission controller prevents users from creating workloads that violate policies you configure in {product-title}.
+Beginning from the {product-title} version 3.0.41, you can also configure the admission controller to prevent updates to workloads that violate policies.
+
+{product-title} uses the `ValidatingAdmissionWebhook` controller to verify that the resource being provisioned complies with the specified security policies.
+To handle this, {product-title} creates a `ValidatingWebhookConfiguration` which contains multiple webhook rules.
+When the Kubernetes or {ocp} API server receives a request that matches one of the webhook rules, the API server sends an `AdmissionReview` request to {product-title}.
+{product-title} then accepts or rejects the request based on the configured security policies.
+
+[NOTE]
+====
+To use admission controller enforcement on {ocp}, you need the {product-title}  version 3.0.49 or newer.
+====
+
+[id="enable-admission-controller-enforcement"]
+== Enabling admission controller enforcement
+
+If you intend to use admission controller enforcement, consider the following:
+
+. *API latency*: Using admission controller enforcement increases Kubernetes  or {ocp} API latency as it involves additional API validation requests.
```

Comment:
* Since this isn't in a particular order, can these be unnumbered?
* Also, as -> because